kytos-init.sh

```diff
@@ -26,6 +26,7 @@ sed -i 's/CONSISTENCY_COOKIE_IGNORED_RANGE =.*/CONSISTENCY_COOKIE_IGNORED_RANGE
 sed -i 's/LIVENESS_DEAD_MULTIPLIER =.*/LIVENESS_DEAD_MULTIPLIER = 3/g' $NAPPS_PATH/var/lib/kytos/napps/kytos/of_lldp/settings.py
 
 # increase logging to facilitate troubleshooting
+test -d /etc/kytos && mv /etc/kytos /etc/kytos-bkp-$(date +%s)
```

Comment:
Was this command meant to be committed or you were only using when developing locally?

Reply:
Not exactly, but I've removed it on commit ca2ba5b. I added it because I was running multiple times the end-to-end tests on the same container and at some point something became weird. Anyway, I agree that more investigation should be done if that behavior happens again and for now I reverted that change! Thanks for point it out!